app/controllers/application_controller.rb

```diff
@@ -1,6 +1,7 @@
 class ApplicationController < ActionController::Base
   protect_from_forgery with: :exception
   before_action :set_locale
+  include SessionsHelper
```

Comment:
giải thích dòng lệnh này nhaaa

Reply:
Ở bên session_helper mình định nghĩa module SessionsHelper cho session ( log in,log out, check log in v...v...) nên mình phải include nó vào trước rồi mới sử dụng được cho sessionscontroller và usercontroller (kế thừa từ app_controller này).